package.json

```diff
@@ -34,14 +35,16 @@
     "lint-staged": "^13.1.0",
     "mailparser": "3.6.3",
     "mkdirp": "1.0.4",
-    "openpgp": "4.10.10",
+    "openpgp": ">=5.5.0",
     "pdfjs-dist": "3.2.146",
     "prettier": "^2.8.3",
     "puppeteer": "19.5.2",
+    "stream-browserify": "^3.0.0",
     "stylelint": "14.16.1",
     "stylelint-config-standard": "29.0.0",
     "typescript": "4.9.4",
-    "web-ext": "7.4.0"
+    "web-ext": "7.4.0",
+    "webpack-cli": "^5.0.1"
```

Comment:
Is `webpack` used somewhere in the project? Haven't found usage in the code

Reply:
I removed the extra packages